tests/5.0/parallel_master_taskloop_simd/test_parallel_master_taskloop_simd.F90

```diff
@@ -0,0 +1,53 @@
+!/===--- test_parallel_master_taskloop_simd.F90 -------------------------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+! This test checks the parallel master taskloop simd directive. The
+! test performs simple operations on an int array which are then
+! checked for correctness.
+!
+!/===----------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_parallel_master_taskloop_simd
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+
+  OMPVV_TEST_VERBOSE(parallel_master_taskloop_simd() .ne. 0)
+
+  OMPVV_REPORT_AND_RETURN()
+
+CONTAINS
+  INTEGER FUNCTION parallel_master_taskloop_simd()
+    INTEGER:: errors = 0
+    INTEGER, DIMENSION(N):: x, y, z
+    INTEGER:: i
+
+    OMPVV_INFOMSG("test_parallel_master_taskloop_simd")
+
+    DO i = 1, N
+       x(i) = 1
+       y(i) = i + 1
+       z(i) = 2*(i + 1)
+    END DO
+
+    !$omp parallel master taskloop simd num_threads(OMPVV_NUM_THREADS_HOST) shared(x, y, z) 
+    DO i = 1, N
+       x(i) = x(i) + y(i) * z(i)
+    END DO
+    !$omp end parallel master taskloop simd
```

Comment:
IMO adding a num_threads check will be useful.

Reply:
Modified both C and Fortran versions to check the number of threads in a parallel region.